presto-hive/src/test/java/com/facebook/presto/hive/statistics/TestMetastoreHiveStatisticsProvider.java

```diff
@@ -56,6 +56,7 @@ public void testGetPartitionsSample()
         assertEquals(getPartitionsSample(ImmutableList.of(p1, p2, p3, p4), 3), getPartitionsSample(ImmutableList.of(p1, p2, p3, p4), 3));
         assertThat(getPartitionsSample(ImmutableList.of(p1, p2, p3, p4), 3)).contains(p1, p4);
         assertEquals(getPartitionsSample(ImmutableList.of(p1, p2, p3, p4, p5), 3).size(), 3);
+        assertThat(getPartitionsSample(ImmutableList.of(p1, p2, p3, p4, p5), 3)).contains(p1, p4, p5);
```

Comment:
@arhimondr With deterministic sampling, this check can be upgraded to equality check.